src/integer_mod_q/modulus/cmp.rs

```diff
@@ -89,3 +159,47 @@ mod test_eq {
         assert_ne!(two, large);
     }
 }
+
```

Comment:
Please include a test for a large modulus and one where you use the same modulus value/object